### ISSUE TITLE
gan training

### ISSUE THREAD

**Issue by 1292765944** [HsinYingLee/DRIT#27]
Did you employ WGAN for training? I notice this [clip_grad_norm_](https://github.com/HsinYingLee/DRIT/blob/master/src/model.py#L201) in your code.

**Comment by hytseng0509** [comment]
We don't employ WGAN in this code. The reason for applying gradient clipping to content discriminator is to avoid explosive gradient and thus stabilize the training.

**Comment by 1292765944** [comment]
@hytseng0509 Yes, the training process seems healthy using gradient clipping. Another question is what could we know with the GAN loss, or say, what should the loss look like if we finish training the model.

**Comment by hytseng0509** [comment]
We usually stop the training when the quality of the translated images stop improving or [G_loss](https://github.com/HsinYingLee/DRIT/blob/709a450f67de593cb7fccae9f342b06d63c98ced/src/model.py#L336) saturates.

**Comment by unlugi** [comment]
Why is gradient clipping applied only for content discriminator gradients?

**Comment by hytseng0509** [comment]
We found it stabilize the training of content discriminator.
